content/en/core/overview/db-schema.md

```diff
@@ -257,12 +257,18 @@ Failed | Task was never terminated and the endDate has past
 Attribute | Description
 -- | --
 user | The user settings id of the user who calculated and created the document. Used for controlling replication. (eg. `org.couchdb.user:agatha`)
-requester | The guid of the contact whose data brought about the creation of the document. Used for controlling cancellation.
+requester | The guid *(globally unique identifier that is an alphanumeric property identifying an object)* of the contact whose data brought about the creation of the document. Used for controlling cancellation. 
```

Comment:
The label `Guid` is used all over the documentation, maybe you can add its definition to the [glossary](https://docs.communityhealthtoolkit.org/glossary/) and then, add links when and where you find necessary.

Reply:
@esthermmoturi done